src/components/core/styles/mixins/selection-panel.scss

```
@@ -0,0 +1,61 @@
+@use './a11y';
+
+@mixin selection-panel-variables {
+  --sbb-selection-expansion-panel-background: var(--sbb-color-white);
+  --sbb-selection-expansion-panel-border-color: var(--sbb-color-cloud);
+  --sbb-selection-expansion-panel-border-radius: var(--sbb-border-radius-4x);
+  --sbb-selection-expansion-panel-border-width: var(--sbb-border-width-1x);
+  --sbb-selection-expansion-panel-input-padding: var(--sbb-spacing-responsive-xs)
+    var(--sbb-spacing-responsive-xxs);
+  --sbb-selection-expansion-panel-animation-duration: var(--sbb-animation-duration-4x);
```

Fallback to --sbb-disable-animation-zero-time needed: 
```suggestion
  --sbb-selection-expansion-panel-animation-duration: var(--sbb-disable-animation-zero-time, var(--sbb-animation-duration-4x));
```